packages/easy-coding-standard/src/Testing/PHPUnit/AbstractCheckerTestCase.php

```diff
@@ -27,7 +27,7 @@
 abstract class AbstractCheckerTestCase extends TestCase implements ConfigAwareInterface
 {
     /**
-     * @var string[]
+     * @var string&literal-string&non-falsy-string[]
      */
     private const POSSIBLE_CODE_SNIFFER_AUTOLOAD_PATHS = [
```

Comment:
@samsonasik Hi, the latest rector dev-main produces this output.
This might be related to https://github.com/rectorphp/rector-src/pull/3022

Reply:
That's seems due to https://github.com/rectorphp/rector-src/pull/3022/files#diff-248cbe18e34821747be287118fa2ae60ea7219a9be2215dfd4e0ab511ac1e801 , I will revert them

Reply:
I see. No rush on this one.

We should have a simple test to cover this in Rector too.

Reply:
Reverted the string specific identifier values on https://github.com/rectorphp/rector-src/pull/3024

Reply:
Let's try again.

Reply:
Rebased, let's try again.

Reply:
Thank you :clap: